packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -3,9 +3,26 @@ import { ContainerAdapterClient } from '../../container_adapter_client'
 import waitForExpect from 'wait-for-expect'
 import { DfTxType, BalanceTransferPayload } from '../../../src/category/account'
 
+const createTokenForContainer = (container: MasterNodeRegTestContainer) => async (address: string, symbol: string, amount: number): Promise<void> => {
```

Comment:
I think we're avoiding arrow functions at the top level

 https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#top-level-functions-should-not-use-arrow-syntax